Data/Sys/GameSettings/R92.ini

```diff
@@ -0,0 +1,14 @@
+# R92E01, R92P01, R92J01 - Pikmin 2 Wii
+[Core]
+# Values set here will override the main Dolphin settings.
+[EmuState]
+# The Emulation State. 1 is worst, 5 is best, 0 is not set.
+EmulationStateId = 4
+EmulationIssues = Distorted sound effects on DSP HLE. Use DSP LLE to fix.
```

Comment:
Is this still true? The new Zelda HLE was merged around when this PR was made, and I'm not sure if you tested before or after that merge.